app/Http/Controllers/CompetitionsController.php

```diff
@@ -3,12 +3,18 @@
 namespace Gladiator\Http\Controllers;
 
 use Illuminate\Http\Request;
-
-use Gladiator\Http\Requests;
-use Gladiator\Http\Controllers\Controller;
+use Gladiator\Models\Competition;
 
 class CompetitionsController extends Controller
 {
+
+    private $validationRules = [
+        'campaign_id' => 'required|numeric',
+        'campaign_run_id' => 'required|numeric',
+        'start_date' => 'required|date',
+        'end_date' => 'required|date',
+    ];
```

Comment:
Might eventually switch this up to use the Form Request class approach thingy, but should be fine for now :)